src/Components/Atoms/CheckBox/index.tsx

```diff
@@ -1,12 +1,12 @@
 import { useState } from 'react'
-import style from './CheckBoxPattern.module.scss'
+import style from './CheckBox.module.scss'
 
 type Props = {
     checked?: boolean
     onChange?: (isChecked: boolean) => void;
 }
 
-export default function CheckBoxPattern({ onChange, checked = false }: Props) {
+export default function CheckBox({ onChange, checked = false }: Props) {
```

Comment:
Checkbox é uma palavra só, trocar em todos os lugares q estão CheckBox 
```suggestion
export default function Checkbox({ onChange, checked = false }: Props) {
```